starter-code/css/main.css

```diff
@@ -0,0 +1,110 @@
+body{
+  margin:5% 20% 5% 4%;
+  font-family: 'Lato', sans-serif;
+}
+
+header h1{
+  font-family: 'Lato', sans-serif;
```

Comment:
Since you set the font-family in the body above, you don't have to reset it here again.